bin/index.js

```diff
@@ -45,6 +45,16 @@ config(function (versions) {
   }
 
   async.doWhilst(getBin, whileCondition, function () {
-    if (switcherBin) return switcher(switcherBin)
+    if (switcherBin) {
+      var switcherProcess = switcher(switcherBin)
+
+      switcherProcess.on('close', function (code) {
+        if (code === 0) {
+          console.log(maxSatisfyVersion)
```

Comment:
I made it in a way that it will not change anything except that there is a line echoed to the terminal with the version of the node used/installed, so it should not change anything for `n` except for this display